make-release.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+    echo "Usage: $0 <release_version>"
+    exit 1
+fi
+
+RELEASE_VERSION=$1
+
+# Update CHANGES.md
+sed -i "s/## Unreleased/## Unreleased\n\n## $RELEASE_VERSION\n/" CHANGES.md
+echo "This description should answer the \"who needs this update?\" question."
+sleep 10
+${EDITOR:-vim} +7 CHANGES.md
+
+# Update version in Dockerfile
+sed -i "s/FROM ghcr\.io\/cnpem\/lnls-debian-epics-7:.*/FROM ghcr.io\/cnpem\/lnls-debian-epics-7:$RELEASE_VERSION AS build-image/" Dockerfile
```

Comment:
This looks like we will still need to keep in sync where version appears.

Can't we do something [similar to the CI job](https://github.com/cnpem/epics-in-docker/pull/98/commits/24101489941fb973b78fb6c51d1775a4caf57181)?